### ISSUE TITLE
Add `Rails.env` to RailsContext

### ISSUE THREAD

**Issue by robwise** [shakacode/react_on_rails#697]
[Heroku Preboot](https://devcenter.heroku.com/articles/preboot) is a feature on Heroku that allows for faster deploy times. When you promote your staging app to production, Preboot simply switches the production server to point at the staging app's container. This means it can deploy much faster since it doesn't have to rebuild anything.

However, this means that if you use the [Define Plugin](https://github.com/webpack/docs/wiki/list-of-plugins#defineplugin) to provide the rails environment to your client code as a variable, that variable will erroneously still have a value of `Staging` instead of `Production`. Providing `Rails.env` (which is accurate even in Preboot) in the default Rails context would be a nice enhancement.

**Comment by justin808** [comment]
@robwise Easy to add, either via a custom Rails Context enhancement (no PR required) or as a default for the RailsContext.

See this example: https://github.com/shakacode/react_on_rails/blob/master/spec/dummy/config/initializers/react_on_rails.rb#L4

```ruby
module RenderingExtension
  # Return a Hash that contains custom values from the view context that will get passed to
  # all calls to react_component and redux_store for rendering
  def self.custom_context(view_context)
    if view_context.controller.is_a?(ActionMailer::Base)
      {}
    else
      {
        somethingUseful: view_context.session[:something_useful]
      }
    end
  end
end
```

**Comment by robwise** [comment]
@justin808 yeah I'm already doing it the custom way, I'm saying I think it should be a default

**Comment by justin808** [comment]
@robwise Maybe for features, we'll do if any of our [Coaching Clients](http://www.shakacode.com/work/shakacode-coaching-plan.pdf) or regular clients want it? Otherwise, we can wait for the PR to come in. Or maybe if the issue gets at least 20 up votes.

**Comment by robwise** [comment]
okay, and maybe just having this issue as a historical record will be good enough in case other people are using preboot and are not realizing that because the webpack isn't being recompiled, the variables may be stale and to watch out for that

**Comment by Conturbo** [comment]
i'm gonna take it

**Comment by justin808** [comment]
Fixed in #1065.
